custom/icds_reports/reports/cas_reach_data.py

```diff
@@ -101,7 +101,7 @@ def get_data_for_daily_usage(date, filters):
                     'help_text': awcs_launched_help_text(),
                     'color': None,
                     'percent': None,
-                    'value': get_value(daily_yesterday if current_month_selected else awc_this_month_data, 'awcs'),
+                    'value': get_value(awc_this_month_data, 'awcs'),
```

Comment:
this column is not more needed you should removed: 
https://github.com/dimagi/commcare-hq/pull/25008/files#diff-50b8c0baaef34ca763dda94410624738R42